user_modules/shared_testcases/commonFunctions.lua

```diff
@@ -4,8 +4,11 @@
   --2. commonFunctions:createString(500) --example
 ---------------------------------------------------------------------------------------------
 local commonFunctions = {}
+require('atf.util')
 local json = require('json4lua/json/json')
-
+require('modules/config')
+local expectations = require('expectations')
```

Comment:
@LitvinenkoIra you need it only in one function, please move this local variables to function